openreview/conference/builder.py

```diff
@@ -1837,7 +1837,7 @@ def get_readers(self, conference, number, review_signature):
         readers = [ conference.get_program_chairs_id()]
 
         if conference.use_area_chairs:
```

Comment:
Not sure why we check if conference uses ACs, since ACs are always the invitees of these invitations

Reply:
Maybe ReviewRating can be used for authors too in venues where there are no ACs. I would leave this for now.